Makefile

```diff
@@ -44,15 +48,22 @@ lint:
 prepare-companion-stack:
 	pytest -v --no-cov integration/setup -m setup
 
-schema:
-	python samtranslator/schema/schema.py > samtranslator/schema/schema.json
+get-cfn-schema:
+	mkdir -p .tmp
+	test -f $(CFN_SCHEMA_PATH) || curl -o $(CFN_SCHEMA_PATH) https://raw.githubusercontent.com/awslabs/goformation/$(CFN_SCHEMA_VERSION)/schema/cloudformation.schema.json
```

Comment:
This feels like a fragile dependency - is there a better, more persistent way to do this?

Reply:
Didn't want to include another 5 MB, but seems fair. And it simplifies upkeep somewhat.